extension/src/popup/views/Account/index.tsx

```diff
@@ -57,6 +58,9 @@ export const Account = () => {
   const { balances, isFunded } = accountBalances;
 
   useEffect(() => {
+    // reset to avoid any residual data eg switching between send and swap or
+    // previous stale sends
+    dispatch(resetSubmission());
```

Comment:
Good call. This is a pretty good place for this. Wonder if this will catch everything 🤔 Let's keep an eye on this, too